config/locales/de_DE.yml

```diff
@@ -71,7 +71,7 @@ de_DE:
         title: Voreinstellung der Aufzeichnungssichtbarkeit
         warning: Diese Einstellung wirkt sich nur auf noch nicht laufende Konferenzen aus.
       registration:
-        info: Verändern der Weise wie sich Teilnehmer auf der Webseite registrieren
+        info: "Methode ändern, wie sich Benutzer auf der Website registrieren"
```

Comment:
Regarding the change of `Webseite` to `Website`: I guess this is mostly personal preference, both versions can be found in Duden.
But in any case I'd keep this consistent with https://github.com/bigbluebutton/greenlight/blob/a6196d7cc92ea36bb09b62ad69c07d688cbe7eb2/config/locales/de_DE.yml#L124 - which currently is still using `Webseite`

Reply:
Webseite/Website:
I learned "Webseite" means a single webpage, "Website" means a hole web presence. So the change would be ok.

*Menu item Site settings*
I changed menu item "Seiteneinstellungen" in Transifex to "Grundeinstellungen". I think "Seiteneinstellungen" is a bad translation of "site settings".
So I also changed "Nutzern mit dieser Rolle erlauben, die primären Webseiteneinstellungen zu ändern"  to "Nutzern mit dieser Rolle erlauben, die Grundeinstellungen zu ändern".